ui/apps/platform/src/Containers/NetworkGraph/NetworkGraphPage.tsx

```diff
@@ -83,7 +83,7 @@ function NetworkGraphPage() {
     } = getScopeHierarchy(searchFilter);
     const simulation = getSimulation(simulationQueryValue);
 
-    const hasClusterNamespaceSelected = Boolean(clusterFromUrl && namespacesFromUrl.length);
+    const hasClusterNamespaceSelected = !!(clusterFromUrl && namespacesFromUrl.length);
```

Comment:
nit: I recently read a compelling argument that casting as a Boolean makes these case easier to pick out when maintaining the code than the old double-negation `!!`

Reply:
ohhh could you share the article??